### PR TITLE
update release notes in plugin.xml

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

**Comment by patflynn** [comment]
@joaoandremartins 
@aslo 
@etanshaul

**Comment by etanshaul** [comment]
I think you should sync the plugin.xml updates with the changelog markdown file

**Comment by etanshaul** [comment]
Oh nevermind looks like it was already updated. LGTM

File: .idea/google-java-format.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="GoogleJavaFormatSettings">
+    <option name="state" value="ENABLED" />
+  </component>
+</project>
```
Comment (joaoandremartins) [inline .idea/google-java-format.xml]: Extra line required

Comment (patflynn) [inline .idea/google-java-format.xml]: sorry I didn't mean to checkin this file.

**Pull request merged**